packages/widget/src/pages/SwapExecutionPage/SwapExecutionPageRouteDetailed.tsx

```diff
@@ -147,9 +152,13 @@ export const SwapExecutionPageRouteDetailed = ({
         : status?.[operation.transferIndex]?.toExplorerLink;
 
       const operationStatus = getOperationStatus(operation);
+      const numberOfTransferEvents = statusData?.transferEvents.length;
+      const latestStatus = status?.[status.length - 1]?.status;
```

Comment:
is this ok to be a key?, because this variable is changing

Reply:
I added these to force it to remount every time these change, that way it will force the loading circles to restart and keep in sync

Reply:
but now that you got me thinking more about it, i can try moving that key to just the loading circle maybe i wonder if that would work too

Reply:
I see